core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -234,15 +240,33 @@ class HttpGoogleServicesDAO(
       }
     }
 
+    def updateGoogleProjectIam(googleProject: GoogleProjectId, policyMap: Map[SamResourcePolicyName, WorkbenchEmail], googleProjectOwnerRole: String, googleProjectViewerRole: String, billingProjectOwnerPolicyEmail: WorkbenchEmail): Future[List[Boolean]] = {
+      // billing project owner - organizations/$ORG_ID/roles/google-project-owner
```

Comment:
Just want to double check that the billing project owner should get permissions here. The project-owner policy grants the (workspace) owner role, but the owner policy group does not include project-owners. That means that Sam sees that project-owners have the workspace owner permissions, but google does not. Therefore, we need to explicitly add the project-owner group here otherwise it wouldn't get the same custom iam role as the owner group.